docs/guides/hybrid-models.md

```diff
@@ -141,7 +141,7 @@ For inference with the `HybridFHEModel` instance, `hybrid_model`, call the regul
 <!--pytest-codeblocks:skip-->
 
 ```python
-hybrid_model.forward(torch.randn((dim, )))
+hybrid_model(torch.randn((dim, )))
 ```
 
 When calling `forward`, the `HybridFHEModel` handles all the necessary intermediate steps for each model part deployed remotely, including:
```

Comment:
if the previous line is changed by removing the `forward`, then you would have to edit this line as well, which starts by

> When calling `forward`